dashboard/config/locales/en.yml

```diff
@@ -709,11 +709,11 @@ en:
       projects_title: 'Open a project'
       secondary_header: 'Ask your teacher what to do next'
   flex_category:
-    required: 'Overview'
+    overview: 'Overview'
```

Comment:
Is this string used before? We need to update those instances, or create a new string.